### PR TITLE
Readd: sidebar

### PR DESCRIPTION
**Pull request by Rashmi-278**

### CODE REVIEW

File: src/pages/upload.tsx [hunk 1]
```diff
@@ -25,14 +25,12 @@ import {
...
   RadioGroup,
   Radio,
-  Spinner,
+  Spinner
 } from "@chakra-ui/react";
 import { providers } from 'ethers'
 import { FaUser } from "react-icons/fa";
 import { useForm } from "react-hook-form";
 import { TextileInstance } from "../services/textile/textile";
-import AddAttributes from "../components/Attributes/AddAttributes";
```
Comment (gawainb) [inline src/pages/upload.tsx:34]: @Rashmi-278 any reason we are removing the metadata attributes?

**Review by gawainb — CHANGES_REQUESTED** [review]
Update on the removal of attributes from NFT metadata?
Metadata standard: 
{
  "description": "Friendly OpenSea Creature that enjoys long swims in the ocean.", 
  "external_url": "https://openseacreatures.io/3", 
  "image": "https://storage.googleapis.com/opensea-prod.appspot.com/puffs/3.png", 
  "name": "Dave Starbelly",
  "attributes": [ ... ], 
}

File: src/pages/upload.tsx (hunk 1 above)
Comment (Rashmi-278) [reply src/pages/upload.tsx]: That was a mistake, fixed it,

**Review by gawainb — APPROVED** [review]
Thank you @Rashmi-278

**Pull request merged**